numba/core/callconv.py

```diff
@@ -572,6 +571,8 @@ def emit_unwrap_dynamic_exception_fn(self, module, st_type, nb_types):
         st_type_ptr = st_type.as_pointer()
         st_ptr = builder.bitcast(fn.args[0], st_type_ptr)
         # compile time values are stored as None
+        # TODO: It should be possible to remove "nb_types" as argument to this
+        # function and get the Numba type from the LLVM type
```

Comment:
```suggestion
```
I'm not sure this is possible as Numba has types that alias to the same LLVM type.